internal/cmprefimpl/cmppg/checks.go

```diff
@@ -502,65 +502,3 @@ func containsOnlyPolygonsOrMultiPolygons(g geom.Geometry) bool {
 		return false
 	}
 }
-
-func checkForceCoordinatesDimension(t *testing.T, want UnaryResult, g geom.Geometry) {
```

Comment:
let me know if still needed.

Reply:
Ahh, this actually is still needed. It was a bug that it wasn't being invoked by anything. Can you add this code back and call it from the list of checks [here](https://github.com/peterstace/simplefeatures/blob/master/internal/cmprefimpl/cmppg/fuzz_test.go#L60)?